python/mxnet/gluon/estimator/estimator.py

```diff
@@ -253,8 +294,14 @@ def fit(self, train_data,
                 for handler in event_handlers:
                     handler.batch_end()
 
-            for metric in self.metrics + self.loss_metrics:
+            if do_validation:
+                self._evaluate(val_data, batch_fn)
+
+            for metric in self.train_metrics + self.train_loss_metrics:
                 self.train_stats['train_' + metric.name].append(metric.get()[1])
+            for metric in self.test_metrics + self.test_loss_metrics:
```

Comment:
Can we rename ```test_metrics``` to ```val_metrics``` for consistency, since we are referring to them as validation stuff throughout

Reply:
we can use `test_metrics` for both validation (at epoch end) and test (at train end) dataset